package.json

```diff
@@ -2,8 +2,9 @@
     "name": "file-regex",
     "version": "3.0.2",
     "description": "Find files by using regex",
-    "main": "index.js",
+    "main": "dist/index.js",
     "scripts": {
+        "postinstall": "npm run build",
```

Comment:
`postinstall` would mean that everyone will have to install `devDependencies`!! which is an unnecessary overhead.
So manually run the build and commit the relevant files.

Reply:
Got it. Removed postinstall. The only thing really necessary to fix the current broken NPM package issue is the change to the "main" property.